packages/browser-wallet/package.json

```diff
@@ -34,6 +34,7 @@
         "react-virtualized-auto-sizer": "^1.0.6",
         "react-window": "^1.8.7",
         "react-window-infinite-loader": "^1.0.8",
+        "stream": "^0.0.2",
```

Comment:
Why is this needed now?

Reply:
I guess we don't get it indirectly from common-sdk or something.

Reply:
Yeah we get it from common-sdk through the api now.